alphagov/paas-cf
# [#179252883] Bump cf deployment to 16.23

What
----

- Bump cf-deployment to 16.23, excluding buildpacks
- Bump cflinuxfs3 to 0.254.0 in line with cf-deployment 16.23
- Bump CAPI to 0.115 in line with cf-deployment 16.23
- Bump bionic-stemcell to 1.25 in line with cf-deployment 16.23
- Address dependabot:
  - Go module miekg/dns in tools/metrics
  - node.js dependencies for error_page
  - Ruby deps addressable and rexml for bundler (omitting rack as required by cf-uaac) 

How to review
-------------

- Code review commit by commit
- Deploy to dev env (See also below)
- Check against acceptance criteria in (https://www.pivotaltracker.com/n/projects/1275640/stories/179252883).

How to deploy
-------------

- Paas-bootstrap stemcells should also be upgraded with this ([PR here](https://github.com/alphagov/paas-bootstrap/pull/472)).

---

🚨⚠️ Please do not merge this pull request via the GitHub UI ⚠️🚨
